### ISSUE TITLE
Deleting a subreddit from 'Subreddit Theme Settings' causes a crash

### ISSUE THREAD

**Issue by Alexendoo** [ccrama/Slide#246]
It prompts if you wish to delete the sub then crashes when you confirm you do. It does however successfully delete the subreddit from the list when going back to check.

```
E/AndroidRuntime(31548): FATAL EXCEPTION: main
E/AndroidRuntime(31548): Process: me.ccrama.redditslide, PID: 31548
E/AndroidRuntime(31548): java.lang.IndexOutOfBoundsException: Invalid index 0, size is 0
E/AndroidRuntime(31548): 	at java.util.ArrayList.throwIndexOutOfBoundsException(ArrayList.java:255)
E/AndroidRuntime(31548): 	at java.util.ArrayList.get(ArrayList.java:308)
E/AndroidRuntime(31548): 	at me.ccrama.redditslide.Adapters.SettingsSubAdapter.getView(SettingsSubAdapter.java:52)
E/AndroidRuntime(31548): 	at android.widget.AbsListView.obtainView(AbsListView.java:2347)
E/AndroidRuntime(31548): 	at android.widget.ListView.makeAndAddView(ListView.java:1864)
E/AndroidRuntime(31548): 	at android.widget.ListView.fillSpecific(ListView.java:1344)
E/AndroidRuntime(31548): 	at android.widget.ListView.layoutChildren(ListView.java:1651)
E/AndroidRuntime(31548): 	at android.widget.AbsListView.onLayout(AbsListView.java:2151)
E/AndroidRuntime(31548): 	at android.view.View.layout(View.java:15671)
E/AndroidRuntime(31548): 	at android.view.ViewGroup.layout(ViewGroup.java:5038)
E/AndroidRuntime(31548): 	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1703)
E/AndroidRuntime(31548): 	at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1557)
E/AndroidRuntime(31548): 	at android.widget.LinearLayout.onLayout(LinearLayout.java:1466)
E/AndroidRuntime(31548): 	at android.view.View.layout(View.java:15671)
E/AndroidRuntime(31548): 	at android.view.ViewGroup.layout(ViewGroup.java:5038)
E/AndroidRuntime(31548): 	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:579)
E/AndroidRuntime(31548): 	at android.widget.FrameLayout.onLayout(FrameLayout.java:514)
E/AndroidRuntime(31548): 	at android.view.View.layout(View.java:15671)
E/AndroidRuntime(31548): 	at android.view.ViewGroup.layout(ViewGroup.java:5038)
E/AndroidRuntime(31548): 	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1703)
E/AndroidRuntime(31548): 	at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1557)
E/AndroidRuntime(31548): 	at android.widget.LinearLayout.onLayout(LinearLayout.java:1466)
E/AndroidRuntime(31548): 	at android.view.View.layout(View.java:15671)
E/AndroidRuntime(31548): 	at android.view.ViewGroup.layout(ViewGroup.java:5038)
E/AndroidRuntime(31548): 	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:579)
E/AndroidRuntime(31548): 	at android.widget.FrameLayout.onLayout(FrameLayout.java:514)
E/AndroidRuntime(31548): 	at android.view.View.layout(View.java:15671)
E/AndroidRuntime(31548): 	at android.view.ViewGroup.layout(ViewGroup.java:5038)
E/AndroidRuntime(31548): 	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1703)
E/AndroidRuntime(31548): 	at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1557)
E/AndroidRuntime(31548): 	at android.widget.LinearLayout.onLayout(LinearLayout.java:1466)
E/AndroidRuntime(31548): 	at android.view.View.layout(View.java:15671)
E/AndroidRuntime(31548): 	at android.view.ViewGroup.layout(ViewGroup.java:5038)
E/AndroidRuntime(31548): 	at me.imid.swipebacklayout.lib.SwipeBackLayout.onLayout(SwipeBackLayout.java:386)
E/AndroidRuntime(31548): 	at android.view.View.layout(View.java:15671)
E/AndroidRuntime(31548): 	at android.view.ViewGroup.layout(ViewGroup.java:5038)
E/AndroidRuntime(31548): 	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:579)
E/AndroidRuntime(31548): 	at android.widget.FrameLayout.onLayout(FrameLayout.java:514)
E/AndroidRuntime(31548): 	at android.view.View.layout(View.java:15671)
E/AndroidRuntime(31548): 	at android.view.ViewGroup.layout(ViewGroup.java:5038)
E/AndroidRuntime(31548): 	at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:2086)
E/AndroidRuntime(31548): 	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:1843)
E/AndroidRuntime(31548): 	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1061)
E/AndroidRuntime(31548): 	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:5885)
E/AndroidRuntime(31548): 	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:767)
E/AndroidRuntime(31548): 	at android.view.Choreographer.doCallbacks(Choreographer.java:580)
E/AndroidRuntime(31548): 	at android.view.Choreographer.doFrame(Choreographer.java:550)
E/AndroidRuntime(31548): 	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:753)
E/AndroidRuntime(31548): 	at android.os.Handler.handleCallback(Handler.java:739)
E/AndroidRuntime(31548): 	at android.os.Handler.dispatchMessage(Handler.java:95)
E/AndroidRuntime(31548): 	at android.os.Looper.loop(Looper.java:135)
E/AndroidRuntime(31548): 	at android.app.ActivityThread.main(ActivityThread.java:5254)
E/AndroidRuntime(31548): 	at java.lang.reflect.Method.invoke(Native Method)
E/AndroidRuntime(31548): 	at java.lang.reflect.Method.invoke(Method.java:372)
E/AndroidRuntime(31548): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:903)
E/AndroidRuntime(31548): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:698)
```

**Comment by ccrama** [comment]
Did you have only 1 sub to begin with?

**Comment by Alexendoo** [comment]
Yeah I was about to test if it was that, but even with two it still crashes

**Comment by ccrama** [comment]
Interesting! Will take a look

**Comment by ccrama** [comment]
Fixed 35c083d5913bf397db49c5b7008fa8bfe1d01c58
